packages/blue-migration-sdk-viem/src/positions/borrow/blue.borrow.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  type ChainId,
+  DEFAULT_SLIPPAGE_TOLERANCE,
+  type Market,
+  type MarketId,
+  MathLib,
+  type Position,
+  getChainAddresses,
+} from "@morpho-org/blue-sdk";
+import type { BlueInputBundlerOperations } from "@morpho-org/bundler-sdk-viem";
+import { maxUint256 } from "viem";
+
+export namespace MigratableBorrowPosition_Blue {
+  export interface Args {
+    marketTo: MarketId;
+    collateralAmount: bigint;
+    borrowAmount: bigint;
+    slippageFrom?: bigint;
+    slippageTo?: bigint;
+  }
+}
+
+export interface MigratableBorrowPositionConfig_Blue {
+  chainId: ChainId;
+  market: Market;
+  position: Pick<Position, "borrowShares" | "user" | "collateral">;
+}
+
+export class MigratableBorrowPosition_Blue
+  implements MigratableBorrowPositionConfig_Blue
+{
+  public readonly market: Market;
+  public readonly position: Pick<
+    Position,
+    "borrowShares" | "user" | "collateral"
+  >;
+  public readonly chainId: ChainId;
```

Comment:
Tip: you don't need to type these when they are typed in the interface (or assigned in the constructor i don't remember)